### PR TITLE
README improvements

### PR DESCRIPTION
**Pull request by phorcys420**
This PR tries to uniformize the README and to make it look better overall.

This PR has the following changes:
- Changes how the badges look.
- Centers a bunch of stuff.
- Adds MarkdownLint to the project as I personally like it, but feel free to yeet it if you don't consider it useful.
- Adds links to other UI libs and links to pub/demo pages.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -108,9 +120,14 @@ See [this](https://github.com/flutter/flutter/issues/46481) for more info on the
 
 See also:
 
-- [Material UI for Flutter](https://flutter.dev/docs/development/ui/widgets/material)
-- [Cupertino UI for Flutter](https://flutter.dev/docs/development/ui/widgets/cupertino)
-- [MacOS UI for Flutter](https://github.com/GroovinChip/macos_ui)
+- [Material UI for Flutter](https://flutter.dev/docs/development/ui/widgets/material) (built-in)
+- [Cupertino UI for Flutter](https://flutter.dev/docs/development/ui/widgets/cupertino) (built-in)
+- [MacOS UI for Flutter](https://github.com/GroovinChip/macos_ui) ([pub](https://pub.dev/packages/macos_ui)) ([demo](https://groovinchip.github.io/macos_ui))
+- [Stockholm (MacOS-like) UI for Flutter](https://github.com/serverpod/stockholm) ([pub](https://pub.dev/packages/stockholm)) ([demo](https://serverpod.dev/stockholm))
```
Comment (bdlukaa) [inline README.md:126]: nah

**Review by bdlukaa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (phorcys420) [reply README.md:126]: do I remove the WIP ones only or the pub and demo links too ?

Comment (bdlukaa) [reply README.md:126]: yup! leave only material and cupertino

Comment (phorcys420) [reply README.md]: Done in [17bc4c3](https://github.com/bdlukaa/fluent_ui/pull/400/commits/17bc4c3fd7476784850387be07bf1989ceeeefa9)

**Review by bdlukaa — APPROVED** [review]

**Pull request merged**